HIRING.md

```diff
@@ -43,7 +43,7 @@ We’re obviously trying to assess the candidate but remember they are assessing
 
 ## Technical Questions
 - What's the superclass of UITableView?
-- Explain how he or she would achieve smooth scrolling in a collection view of 100 000 images
+- Have you faced any challenges regarding achieving 60 frames per second in iOS? Tell us more about it
```

Comment:
No this is waay better. 👌 More stuff like this. Or how they solve simple problems that we all face:

- How do you usually deal with cell configuration?
- How do you deal with retain cycles in closures?